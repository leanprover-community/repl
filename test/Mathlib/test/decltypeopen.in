{"cmd": "import Mathlib.NumberTheory.Divisors\nopen Nat"}

{"cmd": "theorem with_namespace  : (∑ k ∈ Nat.properDivisors 198, k) = 270 := by rfl", "env": 0, "declTypes": true}
