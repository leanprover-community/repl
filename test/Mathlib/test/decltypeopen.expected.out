{"env": 0}

{"messages":
 [{"severity": "info",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 75},
   "data": "Goals accomplished!"}],
 "env": 1,
 "decls":
 [{"type": "∑ k ∈ properDivisors 198, k = 270",
   "pp": "with_namespace",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 22}}]}

